Add newsframent for namespace logger level feature added last year (#64218)
mistercrunch/Airflow: airflow-core/newsfragments/55850.significant.rst

Easily allow setting level of individual loggers

Add ``namespace_levels`` config option (``AIRFLOW__LOGGING__NAMESPACE_LEVELS``) to set log
levels for individual named loggers, independent of the global ``logging_level``.

> This is useful when debugging as, for example it easily lets you enable debug

Reviewer: ```suggestion
This is particularly useful for debugging. For example, it allows enabling debug
```
